app/jobs/algolia/remove_from_index_job.rb

```diff
@@ -0,0 +1,9 @@
+module Algolia
```

Comment:
I'm not sure it's a good idea to namespace this inside the namespace of third party library we don't control. The possibility for conflict in this specific case is next to none but I'd keep our code separated anyway. If for any reason I want to check the module at runtime we don't want the entire Algolia library to appear.

What if we rename this module to Search?

Reply:
ps. make sure Search is free by loading the console and typing it hehe

Reply:
That's a good idea. Though I think `Search` is too general.

Reply:
yeah, but I couldn't think of a better name :(